.github/workflows/ansible-test.yml

```diff
@@ -99,7 +99,9 @@ jobs:
       - name: Set up Python ${{ matrix.ansible }}
```

Comment:
@felixfontein this also uses the wrong ansible version var

Reply:
See #19.